general/device.cpp

```diff
@@ -154,6 +158,17 @@ Device::~Device()
    {
       free(device_option);
 #ifdef MFEM_USE_CEED
+      // Destroy FES -> CeedBasis, CeedElemRestriction hash table contents
+      CeedBasis basis;
+      kh_foreach_value(internal::ceed_basis_from_fes,
+                       basis, CeedBasisDestroy(&basis));
+      kh_destroy(basis, internal::ceed_basis_from_fes);
```

Comment:
Same here -- if these are internal libCEED functions they need to be namespaced.

More importantly, why should a libCEED application even know about them?